index.html

```diff
@@ -8,7 +8,7 @@
     <style>
         body{
             background: orange;
-            border: 10px solid skyblue;
```

Comment:
Greenyellow sucks

Reply:
Be careful, I have access to the same coffee machine as you.